### ISSUE TITLE
[Screen Reader-Help-JavaScript-Call a function]Narrator focus not moving to 'Expand the side documentation' button after pressing enter on 'Collapse the side documentation'button

### ISSUE THREAD

**Issue by v-swasha** [Microsoft/pxt#2543]
**User Experience:**
User who depends on Screen reader will get confused if narrator focus does not retain on 'Expand the side documentation' button after pressing enter on 'Collapse the side documentation' button

**Test Environment**
OS: RS2 Version 1703(OS Build 15063.483)
Platform: Edge.
Screen reader: Narrator

**Repro Steps**
1-Navigate to https://makecode.microbit.org/acc
2-Navigate to the Micro bit section element and select Code control given on it.
3-Navigate to the Help control lying in the header section on the page and select it.
4-Navigate to 'JavaScript' control and select it.
5-Navigate to various controls lying on the pane opened.
6-Navigate to the pane opened and select any link listed under it.
7-Verify narrator focus moving or not on 'Expand the side documentation' button after pressing enter on 'Collapse the side documentation' button.

**Actual Result**
Narrator focus does not retain on 'Expand the side documentation' button after pressing enter on 'Collapse the side documentation' button. After pressing enter on 'Collapse the side documentation' button narrator focus visible on same place not moving to Expand button.

**Expected Result**
Narrator focus should be shift to 'Expand the side documentation' button after pressing enter on 'Collapse the side documentation' button.

**MAS Reference**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={b24201d1-2276-4471-8183-e1e5353031ad}

**Please refer attachment for more details:**
[MAS4.2.10_Call a function_ Narrator focus does not moving to expand side a documentation.zip](https://github.com/Microsoft/pxt/files/1196310/MAS4.2.10_Call.a.function_narrator.focus.does.not.moving.to.expand.side.a.documentation.zip)

**Comment by veler** [comment]
Issue related to the Narrator itself, not the website, we cannot fix it.

**Comment by abchatra** [comment]
External issue with Narrator
